inference/utils/download_llama_weights.py

```diff
@@ -45,6 +45,20 @@ def convert_hf_model(model, dst_folder):
 dst_folder="../weights/llama_7B_weights" if args.use_full_precision else "../weights/llama_7B_weights_half"
 convert_hf_model(model, dst_folder)
 
+# Download and convert model weights only for hf
+
```

Comment:
perfect! Can we make just one small change? I think we should add some parameter to this file to let the user choose which weights to download, or at least have some mechanism to decide whether to download the big models (13b/30b/65b models), since they consume a lot of space on disk. For the CI tests, for example, we should not download them to avoid filling up the instance.

Thanks!